templates/default/node/nodelist.html

```diff
@@ -5,153 +5,159 @@
-		</td>
-		<td scope="col" class="text-right nobr">
-			{t a=$listdata.total}Total: $a{/t}
-		</td>
-	</TR>
-	<TR>
-		<TD colspan="5">
-				<SPAN class="bold">{trans("Filter:")}</SPAN> 
-				<SPAN class="nobr">{trans("Status:")}
-				<SELECT SIZE="1" NAME="s" ONCHANGE="document.choosefilter.submit();">
-					<OPTION VALUE="0"{if $listdata.state == 0} SELECTED{/if}>{trans("- all -")}</OPTION>
-					<OPTION VALUE="1"{if $listdata.state == 1} SELECTED{/if}>{trans("connected<!plural:item>")}</OPTION>
-					<OPTION VALUE="2"{if $listdata.state == 2} SELECTED{/if}>{trans("disconnected<!plural:item>")}</OPTION>
-					<OPTION VALUE="3"{if $listdata.state == 3} SELECTED{/if}>{trans("online")}</OPTION>
-				</SELECT></SPAN>
-				<SPAN class="nobr">{trans("Network:")}
-				<SELECT SIZE="1" NAME="n" ONCHANGE="document.choosefilter.submit();">
-					<OPTION value="0" {if !$listdata.network} SELECTED {/if}>{trans("- all networks -")}</OPTION>
-					{section name=networks loop=$networks}
-					<OPTION value="{$networks[networks].id}" {if $listdata.network == $networks[networks].id} SELECTED {/if}>{$networks[networks].name|truncate:30:"...":true}</OPTION>
-					{/section}
-				</SELECT></SPAN>
-				<SPAN class="nobr">{trans("Group:")}
-				<SELECT SIZE="1" NAME="ng" ONCHANGE="document.choosefilter.submit();">
-					<OPTION value="" {if !$listdata.nodegroup} SELECTED {/if}>{trans("- all groups -")}</OPTION>
-					{foreach from=$nodegroups item=nodegroup}
-					<OPTION value="{$nodegroup.id}" {if $listdata.nodegroup == $nodegroup.id} SELECTED {/if}>{$nodegroup.name|truncate:30:"...":true}</OPTION>
-					{/foreach}
-				</SELECT></SPAN>
-				<SPAN class="nobr">{trans("Customers Group:")}
-				<SELECT SIZE="1" NAME="g" ONCHANGE="document.choosefilter.submit();">
-					<OPTION value="" {if !$listdata.customergroup} SELECTED {/if}>{trans("- all groups -")}</OPTION>
-					{section name=customergroups loop=$customergroups}
-					<OPTION value="{$customergroups[customergroups].id}" {if $listdata.customergroup == $customergroups[customergroups].id} SELECTED {/if}>{$customergroups[customergroups].name|truncate:30:"...":true}</OPTION>
-					{/section}
-				</SELECT></SPAN>
-		</TD>
-	</TR>
-	{if $listdata.total != 0}
-	<TR>
-		<TD class="pagination" colspan="5">
-			{include file="scroller.html" loop=$nodelist}
-		</TD>
-	</TR>
-	{/if}
-	</THEAD>
-	<TFOOT>
-	{if $listdata.total != 0}
-	<TR>
-		<TD class="pagination" colspan="5">
-			{include file="scroller.html" loop=$nodelist}
-		</TD>
-	</TR>
-	{/if}
-	<TR>
-		<TD colspan="3" class="text-right bold">
-			{trans("Total:")}<BR>
-			{trans("Connected<!nodes>:")}<BR>
-			{trans("Disconnected:")}
+	<p style="display: none;">
+		<INPUT type="submit" class="hiddenbtn">
+		<INPUT type="hidden" NAME="m" VALUE="nodelist">
+		<INPUT type="hidden" NAME="page" id="page" VALUE="1">
```

Comment:
poprawia to, bo we wcześniejszym nie było id i skrócony pagination (nawet "stary") nie działał